src/controllers/engine/controllerengine.cpp

```diff
@@ -697,58 +682,55 @@ void ControllerEngine::log(QString message) {
 
 // Purpose: Connect a ControlObject's valueChanged() signal to a script callback function
 // Input:   Control group (e.g. '[Channel1]'), Key name (e.g. 'pfl'), script callback
-// Output:  a ScriptConnectionInvokableWrapper turned into a QtScriptValue.
+// Output:  a ScriptConnectionInvokableWrapper turned into a QJSValue.
 //          The script should store this object to call its
 //          'disconnect' and 'trigger' methods as needed.
 //          If unsuccessful, returns undefined.
-QScriptValue ControllerEngine::makeConnection(QString group, QString name,
-                                              const QScriptValue callback) {
+QJSValue ControllerEngine::makeConnection(QString group, QString name,
+                                              const QJSValue callback) {
     VERIFY_OR_DEBUG_ASSERT(m_pEngine != nullptr) {
         qWarning() << "Tried to connect script callback, but there is no script engine!";
```

Comment:
The qWarning is redundant. The macro already prints to the log that the assumption was violated.

Reply:
Please remove this new commit. VERIFY_OR_DEBUG_ASSERT already takes care of the warning.

Reply:
I just forgot we already talked about this🤦

Reply:
@Pegasus-RPG

Reply:
I'm sorry. I've never yet used verify_or_debug_assert so didn't realize it already logs a failure. Thanks again for your patience with me @ferranpujolcamins !

Reply:
That said, is the plain English text in the removed statement easier to spot/understand during a user troubleshooting session?